pkg/apis/submarinerconfig/v1alpha1/register.go

```diff
@@ -10,19 +10,19 @@ var (
 	GroupName     = "submarineraddon.open-cluster-management.io"
 	GroupVersion  = schema.GroupVersion{Group: GroupName, Version: "v1alpha1"}
 	schemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
-	// Install is a function which adds this version to a scheme
+	// Install is a function which adds this version to a scheme.
 	Install = schemeBuilder.AddToScheme
 
-	// SchemeGroupVersion generated code relies on this name
-	// Deprecated
+	// SchemeGroupVersion generated code relies on this name.
+	// DEPRECATED!
```

Comment:
It's [recommended](https://go.dev/blog/godoc) that deprecated code begin with "Deprecated:" followed by some information about the deprecation.

```suggestion
	// Deprecated: generated code relies on SchemeGroupVersion.
```

Reply:
Yep, that seems a lot more useful than an exclamation mark. :wink: